### PR TITLE
new post: Homebrew's Sigstore-powered provenance is in beta

### PR DESCRIPTION
**Pull request by woodruffw**
This blog post announces the Homebrew provenance beta. 

Some notes:

* The actual link to the ToB blog post is missing, since I don't have it yet. I'll fill it in once I get it 🙂 
* I've marked the date as 5/16, since that's the rough date I was thinking of. However, we can certainly move this around

CC @haydentherapper (I've listed you as the co-author since I expect you'll have suggestions 🙂)

CC @sigstore/core-team

### CODE REVIEW

File: content/homebrew-build-provenance.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title = "Homebrew's Sigstore-powered provenance is in beta"
+date = "2024-05-16"
+tags = ["sigstore", "security", "homebrew"]
+draft = false
+author = "William Woodruff (Trail of Bits), Hayden Blauzvern (Google)"
+type = "post"
++++
+
+Last November, [Alpha-Omega] and [Trail of Bits] announced a
+[collaboration to bring build provenance to homebrew-core].
```
Comment (haydentherapper) [inline content/homebrew-build-provenance.md:11]: Can we link to https://openssf.org/blog/2023/11/06/alpha-omega-grant-to-help-homebrew-reach-slsa-build-level-2/ also?

File: content/homebrew-build-provenance.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Today, we are pleased to announce that the core of that work is live
+and in public beta: [homebrew-core] is now using Sigstore to cryptographically
+attest to all bottles built in the official [Homebrew] CI. This follows
+last year's [npm provenance] feature, making Homebrew the second major
+packaging ecosystem to adopt Sigstore!
```
Comment (haydentherapper) [inline content/homebrew-build-provenance.md:17]: Can we have a line about continuing Sigstore's mission (linking to https://github.com/sigstore/community/blob/main/ROADMAP.md#mission-aka-our-purpose) to support frictionless & transparent provenance on all artifact registries?

**Review by haydentherapper — COMMENTED** [review]
LGTM! Thanks for writing this up!

File: content/homebrew-build-provenance.md (hunk 1 above)
Comment (woodruffw) [reply content/homebrew-build-provenance.md:11]: Yep, I'll work that in 🙂

**Comment by woodruffw** [comment]
I've changed the date here to tomorrow, since that's when we're planning on publishing the longer blog on the ToB side 🙂

**Review by haydentherapper — APPROVED** [review]

**Comment by haydentherapper** [comment]
@bobcallaway @trevrosen @lukehinds @SantiagoTorres @priyawadhwa for approval

File: content/homebrew-build-provenance.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+[active development]: https://github.blog/2024-05-02-introducing-artifact-attestations-now-in-public-beta/
+
+[SLSA Build L2]: https://slsa.dev/spec/v1.0/levels#build-l2
+
+[npm provenance]: https://blog.sigstore.dev/npm-provenance-ga/
+
+[Sigstore's mission]: https://github.com/sigstore/community/blob/main/ROADMAP.md#mission-aka-our-purpose
+
+[Trail of Bits blog]: TODO
```
Comment (woodruffw) [inline content/homebrew-build-provenance.md:63]: I'll update this ASAP.

**Review by SantiagoTorres — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Comment by woodruffw** [comment]
The ToB blog is live and Homebrew has also made a release announcement today, so this is good to go whenever 🙂

**Review by trevrosen — APPROVED** [review]
:shipit:

**Pull request merged**